Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -17,6 +17,8 @@
 static CGFloat const IPadLandscapeWidth = 1024.0f;
 static CGFloat const IPadPro12LandscapeWidth = 1366.0f;
 
+static NSString * const kDefaultEmptyText = @"Nothing to show";
```

Comment:
This string needs to be localised, just use NSLocalizedString() with a an adequate comment, and our translation scripts will pick this up.